server/sonar-server/src/test/java/org/sonar/server/permission/ws/AddUserActionTest.java

```diff
@@ -59,7 +59,18 @@ protected AddUserAction buildWsAction() {
   }
 
   @Test
-  public void add_permission_to_user() throws Exception {
+  public void add_permission_to_user_on_default_organization_if_organization_is_not_specified() throws Exception {
+    loginAsAdmin(db.getDefaultOrganization());
+    newRequest()
+      .setParam(PARAM_USER_LOGIN, user.getLogin())
+      .setParam(PARAM_PERMISSION, SYSTEM_ADMIN)
+      .execute();
+
+    assertThat(db.users().selectGlobalPermissionsOfUser(user, db.getDefaultOrganization())).containsOnly(SYSTEM_ADMIN);
+  }
+
+  @Test
+  public void add_permission_to_user_on_specified_organization() throws Exception {
```

Comment:
This is exactly the same test as `add_permission_to_user_on_default_organization_if_organization_is_not_specified`. The param `organization` seems to miss.